gems/recurly/CVE-2017-0905.yml

```diff
@@ -0,0 +1,28 @@
+---
+gem: recurly
+cve: 2017-0905
+url: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-0905
+date: 2017-11-09
+title: |
+  SSRF vulnerability in Resource#find.
+
+description: |
```

Comment:
Perhaps I should have more information about the vulnerability and the patched versions in this field?

Reply:
I think you can remove the versions from the description since we have a different place for them. Also please format to 80 characters per line